packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -47,6 +47,7 @@
                     <form-field
                         :value="customer.mobileNumber"
                         name="mobile-number"
+                        type="tel"
```

Comment:
We should probably extend f-form-field to accept `:input-type="tel"` rather than sticking the attribute on directly.

Reply:
I agree, although it can be done in a separate ticket, imho.